src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -292,13 +292,13 @@ private IEnumerable<INamespaceSymbol> FilterForFieldsAndProperties(SemanticModel
         {
             var propertySymbols = symbols
                 .OfType<IPropertySymbol>()
-                .Where(property => property.ContainingType?.IsAccessibleWithin(semanticModel.Compilation.Assembly) == true &&
```

Comment:
Bogus product code.  We shouldn't be offer to add an import/using for a member that isn't even accessible to the calling assembly.  

Previously we would allow it if the containing type was still public (but the member wasn't).  This wouldn't help as the using/import still wouldn't fix things.